### ISSUE TITLE
Settings page does not open in 0.8.13

### ISSUE THREAD

**Issue by darthmachina** [jamiebrynes7/obsidian-todoist-plugin#31]
When navigating to the settings page using 0.8.13 (the current insider build) I get a blank page, where on previous versions it worked fine. And the 'Todoist' page title on the left remains highlighted even when navigating away. The app seems to work fine otherwise, but it does seem to me like the settings page is crashing.

I am using the latest 1.2.2 release version of the plugin, and my volcano repo is also up to date.

**Comment by darthmachina** [comment]
FWIW I updated to the latest installer (0.8.12) and insider build (0.8.14) and the issue is still there.

**Comment by jamiebrynes7** [comment]
Hmm sounds like there some breaking changes from `0.8.12` -> `0.8.13`. Are there any errors in the dev console? 

The joys of using an unofficial plugin loader 😂

**Comment by darthmachina** [comment]
Ha, I so rarely work on UI stuff I forget about the dev console sometimes. There is an error:

```
Uncaught TypeError: this.addToggleSetting is not a function
    at Object.fadeAnimationSettings (/home/mike/volcano/p…gins/todoist.js:690)
    at Object.display (/home/mike/volcano/p…gins/todoist.js:685)
    at Object.wl.open (app.js:1)
    at HTMLDivElement.<anonymous> (app.js:1)
```

**Comment by jamiebrynes7** [comment]
Nice, seems like that's changed a bit then! I'll take a look at this and fix it for the next release.

In the meantime, should you need to, you can change the settings manually in `.obsidian/todoist-query-renderer.json`. (You'll probably have to restart Obsidian to get them to take effect though).

**Comment by jamiebrynes7** [comment]
This should be fixed in the next release (`v1.3`)!
